packages/components/src/circular-option-picker/style.scss

```diff
@@ -127,10 +129,22 @@ $color-palette-circle-spacing: 12px;
 }
 
 
-.components-circular-option-picker__dropdown-link-action {
-	margin-right: $grid-unit-20;
+.components-circular-option-picker__dropdown-link-action + .components-button {
+	margin-left: $grid-unit-10;
+}
+
+// Temporarily hide the color card from the post editor, but keep it for Global Styles.
+// @todo: this should be removed and retired entirely when colors in the post editor are
+// able to use the ItemGroup component to handle color options.
+// https://github.com/WordPress/gutenberg/issues/35093
+.edit-post-sidebar .block-editor-panel-color-gradient-settings .components-flex > .components-dropdown {
```

Comment:
I do quite dislike using CSS to ambiguate between global styles and the post editor. But it's a simple way to get this working, and it sits here in CSS that will be mostly retired as we get a chance to revisit this. I'd be happy to be pointed towards a better solution if you can.